src/admin/class-stats.php

```
@@ -153,5 +172,35 @@ private static function format_bytes( $bytes, $decimals = 2 ) {
+			$stats['total_optimized_size'] += $optimized_size / 1024;
+			update_option( self::OPTION_KEY, $stats, false );
+		}
+
+		/**
+		 * Update stats for when an attachment has been restored from bulk optimization.
+		 *
+		 * @param int $attachment_id The attachment ID.
+		 * @param int $optimized_size The optimized file size in bytes.
+		 * @param int $restored_size The restored file size in bytes.
+		 */
+		public static function update_stats_bulk_restored( $attachment_id, $optimized_size, $restored_size ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attachment_id'. (undefined)

(UnusedFormalParameter)